<!DOCTYPE html>
<html>
   
   <head>
    <title>1st International Workshop on Novel Data Mining Methods for the Analysis of High-Throughput Biological Data (DM-HT-D)</title>
   </head>
      <hr width="10000">
      <h1>Held in Conjunction wiht the 20th IEEE International Conference on Data Mining  IEEE ICDM 2020
Sorrento, Italy, November 17-20, 2020 <br>
http://icdm2020.bigke.org/</h1>


   
<body>
<p>The development of High-Throughout (HT) innovative technologies (e.g., next-generation sequencing, microarray, and mass spectrometry)
for the analysis of biological samples as for instance, genomics, transcriptomics, proteomics and metabolomics, led to generate tera or 
even peta-Byte sized files storing heterogeneous and complex data of several organisms among which human. 
Research in this area is data-intensive; that is, data sets are massive and highly heterogeneous.
To produce knowledge from this data, researchers need to integrate these large and diverse data sets. The integration of different data 
can contribute to obtaining a holistic vision of complex phenomena such as cancer or diabetes, impossible to achieve by analyzing the data 
individually. 
As a result, HT data need for different and more efficient data mining techniques, starting from data preprocessing, transformation, and 
knowledge extraction methods, since the traditional data mining methods are neither effective nor efficient. Even if conventional data 
mining methods may be adapted for analyzing HT data, they do not guarantee enough optimization in response times, resource management, 
as well as the obtained results, may present low accuracy, and quality. Thus, it is mandatory to develop new techniques that can help 
the researcher to spur light in the behavior of complex biological phenomena such as cancer, diabetes, and Alzheimer.
Nevertheless, the development of new methodologies is challenging because of the large scale, heterogeneity, and complexity of the current 
HT experimental data sets. 
New supervised or unsupervised data mining algorithms should be defined by taking into account the constraints mentioned above, 
to provide methods for the efficient extraction of biological meaningful information from the large-scale amount of life science data. 
Lately, the application of such techniques applied to personalized and translational medicine is becoming a central area. As methodologies
evolve, data mining analysis has the potential to discover the complexity of molecular mechanisms governing complex diseases as well as 
to drive researchers to develop new drugs by revolutionizing the process of drug discovery.
Many research problems in current HT data are still open, such as the integration of multiple databases, which is related to the 
different data representation, which makes it challenging to use in the same experimental data coming from several databases. 
To simplify the researcher's work, it is mandatory to develop automatic preprocessing methods to provide a consistent data representation, 
allowing a broader view of biological events not available without a uniform data representation. Moreover, the combined use of data mining
and text mining can further improve the analysis of heterogeneous sources (i.e., protein, DNA, RNA, and microRNA), which are emerging 
research topics.
This workshop aims to bring together researchers in data mining, bioinformatics, and biomedicine to discuss the current state of the art, 
by using data mining to discover biological knowledge, focusing mainly on applications on biomedicine, omics science, drug discovery, 
sharing and improving the best practice guidelines for reporting the fundamental principles of data mining theory in life sciences, 
representing an opportunity to facilitate interdisciplinary collaborations.</p>

<h2>Topics</h2>
Topics of interest include, but not limited to:<br>
Data mining<br>
Genomics<br>
Proteomics<br>
Interactomics<br>
Artificial Intelligence<br>
Metabolomics<br>
Text Mining<br>
Deep Learning<br>
Network Analysis<br>

<h2>PROGRAM</h2>
The workshop will take place on November 17-20, 2020, Sorrento, Italy (To Be Announced). The program is not available yet.<br>
Submission Guidelines<br>
The paper submission site:<br>
https://wi-lab.com/cyberchair/2020/icdm20/scripts/ws_submit.php?subarea=S <br>

Paper submissions should be limited to a maximum of ten (10) pages, in the IEEE 2-column format (https://www.ieee.org/conferences/publishing/templates.html),<br> 
including the bibliography and any possible appendices. Submissions longer than 10 pages will be rejected without review. <br> 
All submissions will be triple-blind reviewed by the Program Committee on the basis of technical quality, relevance to scope of the conference, 
originality, significance, and clarity. The following sections give further information for authors.<br>

   <h2>Triple blind submission guidelines</h2>
Since 2011, ICDM has imposed a triple blind submission and review policy for all submissions. Authors must hence not use 
identifying information in the text of the paper and bibliographies must be referenced to preserve anonymity. 
Any papers available on the Web (including Arxiv) no longer qualify for ICDM submissions, as their author information is already public.
Accepted papers will be published in the conference proceedings by the IEEE Computer Society Press. All manuscripts are submitted as full papers and are reviewed based on their scientific merit. There is no separate abstract submission step. There are no separate industrial, application, short paper or poster tracks during submission. Manuscripts must be submitted electronically in online submission system (http://wi-lab.com/cyberchair/2020/icdm20/scripts/submit.php?subarea=DM). We do not accept email submissions.
   <h2>JOURNAL SPECIAL ISSUE</h2>
A selected number of best papers will be invited for possible inclusion, in an expanded and revised form, <br>
in the Knowledge and Information Systems journal (http://kais.bigke.org/) published by Springer.<br>

   <h2>IMPORTANT DATES </h2>
Full paper submissions: June 2, 2020 <br>
Contest requirement specification and sample data available: June 12, 2020<br>
Contest team registration begins. Data sets are available: June 26, 2020<br>
Demo and tutorial proposals: July 10, 2020<br>
Contest final submission deadline: 23:59, July 31, 2020.<br>
Workshop paper submissions: August 24, 2020<br>
Conference paper, tutorial, demo notification: August 20, 2020<br>
Contest notifications to shortlisted teams for web applications: 23:59, August 10, 2020<br>
Contest finalist notifications: August 28, 2020<br>
Workshop paper notification: September 17, 2020<br>
Camera-ready deadline and copyright forms: September 24, 2020<br>
Conference dates: November 17-20, 2020<br>
Contest prize presentations at ICDM 2020: November 19, 2020<br>
More Information
More information about ICDM 2020: http://icdm2020.bigke.org/<br>

   <h2>Organizer</h2><br>
Giuseppe Agapito, University Magna Graecia of Catanzaro, Italy<br>
Chiara Zucco, University Magna Graecia of Catanzaro, Italy<br>

   <h2>PROGRAM COMMITTEE</h2><br>
Marianna Milano, University Magna Graecia of Catanzaro, Italy<br>
Marzia Settino, University Magna Graecia of Catanzaro, Italy<br>
Barbara Calabrese, University Magna Graecia of Catanzaro, Italy<br>
Chiara Pastrello Princess Margaret Cancer Center, UHN, Toronto, Canada<br>
Alberto Falcone, Univeristy of Calabria, Italy<br>
Antonio Guerriero, Icar-CNR, Italy<br>
Claudio Savaglio, Univeristy of Calabria, Italy<br>
Loris Belcastro, Univeristy of Calabria, Italy<br>
Mario Cannataro, University Magna Graecia of Catanzaro, Italy<br>

<h2>Contacts:</h2><br>
Giuseppe Agapito<br>
Chiara Zucco<br>

</body>
</html>
